src/overlays/actors/ovl_Obj_Mure2/z_obj_mure2.c

```diff
@@ -10,14 +10,35 @@
 
 #define THIS ((ObjMure2*)thisx)
 
+#define OBJ_MURE2_GET_CHILD_TYPE(thisx) ((thisx)->params & 3)
+
+#define OBJ_MURE2_CHILD_COUNT_BUSH_RING 9
+#define OBJ_MURE2_CHILD_COUNT_BUSH_SCATTERED 12
+#define OBJ_MURE2_CHILD_COUNT_ROCK_RING 8
+
+typedef enum Mure2ChildType {
+    OBJMURE2_CHILDTYPE_BUSH_RING,
+    OBJMURE2_CHILDTYPE_BUSH_SCATTERED,
+    OBJMURE2_CHILDTYPE_ROCK_RING
```

Comment:
```suggestion
    /* 0 */ OBJMURE2_CHILDTYPE_BUSH_RING,
    /* 1 */ OBJMURE2_CHILDTYPE_BUSH_SCATTERED,
    /* 2 */ OBJMURE2_CHILDTYPE_ROCK_RING
```

Reply:
A `_MAX` would be nice too, so it can be used as the size for all the arrays that are indexed with this enum